forum/qa-plugin/outdated-question-info/frontend/show-outdated-question-info.js

```diff
@@ -18,16 +18,16 @@ if(placeOfOutdatedQuestionInfo){
     if(questionElemExist){
         const outdatedInfoContainerClassList = questionElemExist.classList;
     
-        const areThereAnswersForQuestion = !CKEDITOR.instances.a_content ? true : false;
+        const areThereAnswersForQuestion = !CKEDITOR.instances.a_content;
         
 
 
         document.querySelector('#q_doanswer').addEventListener('click', ()=>{
-            outdatedInfoContainerClassList.contains('hidden') ? outdatedInfoContainerClassList.remove('hidden') : outdatedInfoContainerClassList.add('hidden');
+            outdatedInfoContainerClassList.toggle('hidden');
```

Comment:
Nie `.toggle('hidden')` tylko `.remove('hidden')` - eksplicytnie usuwamy klasę (pokazując tym samym komunikat) na kliknięcie w przycisk dodający odpowiedź. Tak jak pokazałem [w komentarzu](https://github.com/CodersCommunity/forum.pasja-informatyki.local/pull/297#discussion_r900323159).

Reply:
Nie jestem pewny czy to mogłoby zadziałać. Formularz odpowiedzi możemy zamknąć również przyciskiem otwierania odpowiedzi, a nie tylko czerwonym anuluj. Toggle jest tu aby nie było jasno narzucone, którym przyciskiem mamy zamykać formularz

Reply:
Faktycznie, przycisk "Odpowiedz" toggluje formularz z edytorem - nie zauważyłem. `.toggle('hidden')` może zostać.